dataverse-webapp/src/main/webapp/editdatafiles.xhtml

```diff
@@ -60,10 +57,17 @@
 
                     <ui:include src="editFilesFragment.xhtml">
                         <ui:param name="editDatafilesPage" value="true"/>
+                        <ui:param name="datasetPage" value="false"/>
```

Comment:
I'm not sure what does this parameter mean, when should it be false/true? What does it control?

Reply:
I've added explicit parameter to have it coupled with the one on dataset.xhtml. During separation of `Files` tab we'll have explicit information that section rendered with `datasetPage` could be removed from editdatafiles.xhtml perspective.
It serves the same purpose as `editDatafilesPage` parameter.

Reply:
Maybe it should be named something like `renderDatasetPageSection`?

Reply:
Everywhere in the app we use `datasetPage` to reference anything that happens on/comes from dataset.xhmt.
I would leave it as it is.

Reply:
If something is badly named when is it the time to change it if not now?

Reply:
This param will be removed on separation from dataset.xhtml.
But if you insist then no problem.